CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Added
+
+- Implement metrics for grafanaOrganizations.
```

Comment:
Can you bé more explicit on the metric?

Reply:
Is that not like 3 metrics only? But I feel like you should probably explain a bit more than implement metrics for grafana organizations :D

Reply:
> Is that not like 3 metrics only?

Well, it'll depend on how much we'll decide to keep :shrug: